### PR TITLE
feat(contact): add contact form

### PR DESCRIPTION
**Pull request by l-bonanni**

### CODE REVIEW

File: components/Textarea.vue [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+<script>
+export default {
+  props: ["placeholder"]
+};
+</script>
+
+<style scoped lang="scss">
+textarea {
+  box-sizing: border-box;
```
Comment (mfrachet) [inline components/Textarea.vue]: the size of the textarea is really small by default. Can't we augment the size?

File: components/Radio.vue [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+};
+</script>
+
+<style scoped land="scss">
+.radio {
+  display: inline-block;
+  margin-bottom: 1em;
+}
+
+.radio + .radio {
```
Comment (mfrachet) [inline components/Radio.vue]: `+` ?

File: components/modules/Contact.vue [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+                <app-input :placeholder="$t('contact.name-label')"></app-input>
+                <app-input :placeholder="$t('contact.mail-label')"></app-input>
+              </div>
+              <app-textarea :placeholder="$t('contact.message-label')"></app-textarea>
+              <app-button>{{$t('contact.button-label')}}</app-button>
+            </form>
+          </app-col>
+          <app-col size="1" id="contact-info">
+            <img src="/zenika.png" alt="Zenika" />
+            <div><strong>{{$t('contact.mail-label')}} :</strong><br>{{$t('contact.mail')}}</div>
```
Comment (mfrachet) [inline components/modules/Contact.vue]: `<br>` 🙄 ?

File: components/modules/Contact.vue [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+  & > input {
+    float: left;
+
+    & + input {
+      float: right;
+    }
+  }
+
+  input {
+    width: calc(50% - 0.5em);
```
Comment (mfrachet) [inline components/modules/Contact.vue]: what is the .5 em ?

File: i18n/en.json [hunk 5]
```diff
@@ -9,9 +9,19 @@
   },
   "keywords": "Agile,Web,Craft,Java",
   "contact": {
+    "title": "Contact",
+    "subtitle": "Question ? Leave us a message :",
```
Comment (mfrachet) [inline i18n/en.json]: En anglais, y'a pas de space avant le "?"

File: components/modules/Contact.vue (hunk 4 above)
Comment (l-bonanni) [reply components/modules/Contact.vue]: @mfrachet i want a space of 1rem between each input, so it's this size divided by two

File: components/modules/Contact.vue (hunk 3 above)
Comment (l-bonanni) [reply components/modules/Contact.vue]: @mfrachet line break :)

File: components/Radio.vue (hunk 2 above)
Comment (l-bonanni) [reply components/Radio.vue]: @mfrachet it means "his direct brother"

**Pull request merged**